### DISCUSSION TITLE
求一个豆瓣读书的app

### DISCUSSION THREAD

**Discussion by ch0c00late** [tw93/Pake#358] [Q&A]
平时整理手头的电子书，会用到豆瓣读书https://book.douban.com/的网站，主要用来输入ISBN编码或者书名来获取书籍信息，然后我把手上的电子书根据“ISBN编码+书名”的形式进行存储。edge提供的网页封装应用感觉比较一般，希望作者考虑。

**Comment by Tlntin** [comment]
可以使用在线打包，参考该[教程](https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89)
